tests/all-spark-notebook/test_spark_notebooks.py

```diff
@@ -38,8 +38,8 @@ def test_nbconvert(container: TrackedContainer, test_file: str) -> None:
         command=["start.sh", "bash", "-c", command],
     )
     warnings = TrackedContainer.get_warnings(logs)
-    # Some Spark warnings
-    assert len(warnings) == 5
+    # No Spark warnings
+    assert len(warnings) == 0
```

Comment:
Just remove `no_warnings=False,` line and all lines about the warnings.

Reply:
Done.